### ISSUE TITLE
build 240 and 241 very slow with HVSC DB enabled

### ISSUE THREAD

**Issue by Sebby75** [droidmjt/Droidsound#606]
Updated this morning to build 241 (64bit) and immediately started having issues with app being not responsive while playing sid tunes. Previously used build 239 for 6 weeks almost daily without any problems..

Issue seem to have something to do with HVSC/STIL song info database handling. When HVSC DB is enabled app slows down a lot.. When song is selected it takes a couple of seconds to switch from Browser to Player and even after it switches if you slide between browser/player/search while sid tune with song description is playing there is very significant slowdown (to the point where I get "app not responding" popup asking me to wait or close). Video below should clearly show the problem..

https://www.youtube.com/watch?v=LSAuaRm7hzM

Also tested build 240 and it's exactly same... Reverted to 239 for now which doesn't have this issue.
Sidplay2fp plugin seem to have been updated from 2.0.4 to 2.0.5 between 239 and 240 build (I don't know what else might have changed).

Only tested on HUAWEI P20 Pro

Thanks
Seb

**Comment by droidmjt** [comment]
I noticed this slowness after moving to version 2.0.5. Nothing else has been changed for sidplay2fp plugin. I'll see if I can get it fixed.

**Comment by Sebby75** [comment]
Fixed now.. Working lovely again. Thank you for looking into it.

https://www.youtube.com/watch?v=vhXpInk7O5w

Whatever that bug was, looks like it affected Huawei devices the most... I have tested build 241 on couple of devices. Motorola RAZRi from 2012 (this is single core Intel Atom Z2460), Samsung Galaxy Tab S2 from 2016 (Snapdragon 652), Huawei P20 Pro (Kirin 970) and Huawei P30 Pro (Kirin 980). Slowdown on both Huawei devices was most pronounced, and almost non-existent on Samsung Galaxy Tab S2, even that 8 year old ancient Motorola wasn't that much affected.

Kind regards
Seb
